operations/node_management/Rebuild_NCNs/Re-add_Storage_Node_to_Ceph.md

```diff
@@ -189,7 +183,7 @@ Use the following procedure to re-add a Ceph node to the Ceph cluster.
 1. Add nodes into HAproxy and KeepAlived.
 
    ```bash
-   pdsh -w ncn-s00[1..(end node number)] -f 2 '/srv/cray/scripts/metal/generate_haproxy_cfg.sh; systemctl restart haproxy.service; /srv/cray/scripts/metal/generate_keepalived_conf.sh; systemctl restart keepalived.service'
+   pdsh -w ncn-s00[1-(end node number)] -f 2 'source /srv/cray/scripts/metal/update_apparmor.sh ; reconfigure-apparmor; /srv/cray/scripts/metal/generate_haproxy_cfg.sh > /etc/haproxy/haproxy.cfg; systemctl enable haproxy.service; systemctl restart haproxy.service; /srv/cray/scripts/metal/generate_keepalived_conf.sh > /etc/keepalived/keepalived.conf; systemctl enable keepalived.service; systemctl restart keepalived.service'
```

Comment:
Can the pdsh command update to hit all the storage nodes or is that not appropriate?

Reply:
good question -- only the admin knows which nodes should be running RGW -- so we leave that to them..